jte-jsp-converter/src/test/java/gg/jte/convert/jsp/JspToJteConverterTest.java

```diff
@@ -244,12 +251,17 @@ void givenSetup(Consumer<Converter> setup) {
         setups.add(setup);
     }
 
-    void whenJspTagIsConverted(String jspTag, String jteTag) {
+    void whenJspTagIsConverted(String jspTag, String jteTag, boolean putParametersOnSeparateLines) {
```

Comment:
Do we need a parameter for this? I think its a sane default to format the parameters like you did. As @kelunik pointed out if someone requires different formatting, this should be done through a formatter. Conversion is a one time job, the need to format jte files will come after that many more times.

Right now you test only that this usecase works. All other conversion tests might work, but we're not sure and we don't know how they look like with this parameter on.